setup.cfg

```
@@ -34,12 +34,13 @@ classifiers =
     Programming Language :: Python :: 3.10
     Programming Language :: Python :: 3.11
     Programming Language :: Python :: 3.12
+    Programming Language :: Python :: 3.13
     Topic :: Software Development :: Libraries
 
 [options]
 zip_safe = True
 setup_requires = setuptools_scm
-install_requires = six >= 1.6
+install_requires = six >= 1.17
```

Only *need* min 1.16:
```suggestion
install_requires = six >= 1.16
```